packages/mobile/src/navigator/Headers.tsx

```diff
@@ -103,6 +104,20 @@ export const headerWithBackButton: StackNavigationOptions = {
   headerLeft: ({ canGoBack }) => (canGoBack ? <BackButton /> : null),
 }
 
+export function headerWithBackButtonEvent({
```

Comment:
wouldn't it make more sense to just put the `eventName` prop in `headerWithBackButton` since it's optional anyways?

Reply:
Yeah, we could change the `headerWithBackButton` (and `headerWithCloseButton`) to take an optional `eventName` param; I considered that, but wasn't sure if we should modify it since it's currently being used in a bunch of different places and would require changes to a bunch of files to accommodate that change.